decidim-core/spec/uploaders/application_uploader_spec.rb

```diff
@@ -79,49 +83,45 @@ module Decidim
       end
 
       context "with production environment" do
+        let(:hostname) { "production.decidim.org" }
```

Comment:
I prefer to use the `example.org` domain (`production.example.org`) or the `.test` TLD (`production.decidim.test`)  for this kind of specs. As I see that [we're already using example.org with cdn.example.org](
https://github.com/mainio/decidim/blob/33bbf077fa36d61c77a68349b04ad1ea83f72431/decidim-core/spec/uploaders/application_uploader_spec.rb#L131) I think we should use `production.decidim.test` so it's clear that's different from the CDN server.